datasets/GazeBase/stimulus_TEX_R1S1_bg/trial_1_participant_100_TEX_R1S1_bg.json

```diff
@@ -1,5 +1,356 @@
 {
+    "time_stamps": [
+        505.0,
```

Comment:
timestamps should not repeat.  Also, there's a huge gap between the second and 3rd fixation.  This can't be right, saccades are never that long.